packages/terra-framework-docs/src/terra-dev-site/doc/worklist-data-grid/example/DefaultWorklistDataGrid.jsx

```diff
@@ -1,22 +1,98 @@
-import React from 'react';
+import React, { useRef, useState } from 'react';
 import WorklistDataGrid from 'terra-worklist-data-grid';
 import gridDataJSON from './gridData.json';
 
 const DefaultWorklistDataGrid = () => {
+  const rowSelectionModeRef = useRef();
   const rowHeaderIndex = 0;
   const { cols, rows } = gridDataJSON;
+  const [selectedRows, setSelectedRows] = useState([]);
+  const [hasSelectableRows, setHasSelectableRows] = useState(false);
```

Comment:
This should not be a state variable.  The consumer would be the one to set the prop to disable row selection mode.  It would not be handled internally.  The WorklistDataGrid should simply be notifying the parent that the mode has been requested to be changed.